tests/stub/routing/scripts/v3_no_routing/optional_hello.script

```diff
@@ -1,6 +1,7 @@
 !: BOLT #VERSION#
-!: AUTO RESET
-!: AUTO GOODBYE
 
+?: GOODBYE
```

Comment:
Is this expected here?

Reply:
JS seems a bit rude and doesn't even say `HELLO` to figure out if the server has multi db support. Arguable it should because unauthenticated connections will be terminated after a while on a real server, if I'm not mistaken. @bigmontz thoughts on this?